src/instance.ts

```diff
@@ -41,7 +129,13 @@ import {Database} from './database';
  * const instance = spanner.instance('my-instance');
  */
 class Instance extends common.ServiceObject {
-  constructor(spanner, name) {
+  formattedName_: string;
+  // tslint:disable-next-line: no-any
+  request: (config: any, callback: Function) => void;
```

Comment:
Maybe we should add a types for `config` and `callback` inside of the index file.